gwcs/wcs.py

```diff
@@ -334,6 +334,63 @@ def _invert(self, *args, **kwargs):
         """
         raise NotImplementedError
 
+    def in_image(self, *args, **kwargs):
+        """
+        This method tests if one or more of the input world coordinates are
+        contained within forward transformation's image and that it maps to
+        the domain of definition of the forward transformation.
+        In practical terms, this function tests
+        that input world coordinate(s) can be converted to input frame and that
+        it is within the forward transformation's ``bounding_box`` when
+        defined.
+
+        Parameters
+        ----------
+        args : float, array like, `~astropy.coordinates.SkyCoord` or
+            `~astropy.units.Unit` coordinates to be inverted
```

Comment:
Did you mean `~astropy.units.Quantity`, not `Unit` here? 
And does this work for mixed spatial and spectral?

Reply:
@nden This line(s) were "borrowed" from the docstring for `invert()`. Should that one fixed as well?

Reply:
@nden Tested on a MIRI image with mixed spatial and spectral WCS. Seems to work as expected.